src/plugin/debugger/mhpdbgc.c

```diff
@@ -1351,6 +1377,28 @@ static void mhp_mcbs(int argc, char *argv[])
       hdr = 1;
     }
   }
+
+  // Now see if HMA is in Win95 MCB format
```

Comment:
Would you like to rev-engineer
fn 4a04 a little to find out where
does it get the start address from?
I bet its somewhere in LoL or in
some other struct, from were we
could get it too.

Reply:
Mmm, looks hardcoded to me
~~~
dosdebug> 
system state: stopped
AX=0000  BX=073e  CX=000d  DX=2074  SI=448b  DI=3608  SP=feaa  BP=0e80
DS=0040  ES=a136  FS=0000  GS=0000  FL=000a3346
CS:IP=ffff:231c       SS:SP=082d:feaa

ffff:231c 0E               push cs

dosdebug> 
system state: stopped
AX=0000  BX=073e  CX=000d  DX=2074  SI=448b  DI=3608  SP=fea8  BP=0e80
DS=0040  ES=a136  FS=0000  GS=0000  FL=000a3346
CS:IP=ffff:231d       SS:SP=082d:fea8

ffff:231d 07               pop  es

dosdebug> 
system state: stopped
AX=0000  BX=073e  CX=000d  DX=2074  SI=448b  DI=3608  SP=feaa  BP=0e80
DS=0040  ES=ffff  FS=0000  GS=0000  FL=000a3346
CS:IP=ffff:231e       SS:SP=082d:feaa

ffff:231e BF3000           mov  di,0030
~~~

Reply:
Such a small offset?
Any idea what are the first
32 bytes of HMA are?
Maybe there is also some
unadvertised mcb?

As for scanning I just worry
that it may reveal some "internal"
mcb, i.e. DOS data probably has
a copy of mcb within, from which
it populates the real ones... and
you may find that.
But since its hard-coded, perhaps
there is no way around.

Reply:
I suppose we could always call int2f/4a04 early on and and populate a global variable in a similar fashion to how we get the lol value?

Reply:
Sounds good!